src/components/Pages/Details.jsx

```diff
@@ -7,6 +8,7 @@ import NavBar from '../props/NavBar';
 import Footer from '../props/Footer';
 import back from '../../imgs/back.png';
 import Loading from '../../imgs/loading.gif';
+import '../../styles/details.css';
 
 function Details() {
```

Comment:
- [ ] As you have been requested of you by the previous reviewer, it is mandatory that your submitted design looks alike to the one provided in the Behance design. Tis included themes, colors, typography etc. This said, kindly ensure that your Details page design looks like the one that is provided. This should be a "Zebra crossing effect"

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/204353667-f83b56a5-797a-4c93-ad2a-7f20a964de4a.png)


**This is the desired design**

![image](https://user-images.githubusercontent.com/86318284/204353564-0ce97eb2-28c8-4a0e-9c04-230c37d8cab9.png)

Reply:
The design has been updated accordingly.